src/sndfile.c

```diff
@@ -1421,13 +1421,19 @@ sf_command	(SNDFILE *sndfile, int command, void *data, int datasize)
 
 		case SFC_GET_CHANNEL_MAP_INFO :
 			if (psf->channel_map == NULL)
-				return SF_FALSE ;
+				return SF_FALSE;
```

Comment:
Formatting error, please fix. This project leaves a space before the final semicolon.